utils/bump_version.py

```diff
@@ -87,7 +91,7 @@ def processing(message):
 
 
 class Changes:
-    def __init__(self, path):
+    def __init__(self, path: Path):
```

Comment:
I personally don't like half-typed methods.

```suggestion
    def __init__(self, path: Path) -> None:
```